src/app/clusters/on-off-server/on-off-server.cpp

```diff
@@ -518,6 +539,8 @@ bool OnOffServer::OnWithTimedOffCommand(const app::ConcreteCommandPath & command
  */
 void OnOffServer::updateOnOffTimeCommand(chip::EndpointId endpoint)
 {
+    const chip::System::Clock::Timestamp enterTime = chip::System::SystemClock().GetMonotonicTimestamp();
```

Comment:
Wouldn't we really want to compensate here for the timer running late compared to when we planned to set it for, not just whatever processing delay this function itself incurs?

Reply:
true, in this particular place we would not really need to compensate